File: Klijent/Arena.js
//import { Organizacija } from "./Organizacija";
export class Arena{

    constructor(id,ime, drzava,kapacitet){
   
        this.id = id
        this.ime = ime
        this.drzava=drzava
        this.kapacitet = kapacitet;
     
    }


crtajtabelicuArene(host)
{
    var tr=document.createElement("tr");
    host.appendChild(tr);


var el=document.createElement("td");
el.innerHTML=this.id;
tr.appendChild(el);
el=document.createElement("td");
el.innerHTML=this.ime;
tr.appendChild(el);
el=document.createElement("td");
el.innerHTML=this.drzava;
tr.appendChild(el);
el=document.createElement("td");
el.innerHTML=this.kapacitet;
tr.appendChild(el);
}

}

File: Models/Arena.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Models{

[Table("Arena")]
public class Arena{


[Key]
public int ID { get; set; }

[Required]
[MaxLength(20)]
public string Ime { get; set; }

[Required]
[MaxLength(20)]
public string Drzava { get; set; }

[Required]
[Range(1000,100000)]
public int Kapacitet { get; set; }

[Required]
public bool PotrebnaCovidPropusnica { get; set; }

[JsonIgnore]
public List<Borba> Borbe{get;set;}

public Organizacija Organizacija {get;set;}


}
}